src/server/scripts/Northrend/zone_zuldrak.cpp

```diff
@@ -587,7 +588,7 @@ struct npc_overlord_drakuru_betrayal : public ScriptedAI
                     lich->GetMotionMaster()->MovePoint(0, 6141.2393, -2011.2728, 589.8653);
                 break;
             case EVENT_BETRAYAL_EPILOGUE_10:
-                EnterEvadeMode(EVADE_REASON_OTHER);
+                Reset();
```

Comment:
```suggestion
                _state = BETRAYAL_EVADE;
                EnterEvadeMode(EVADE_REASON_OTHER);
                
```

same behavior, but more consistent with manually handling Evade